docs/contribute/contributing.md

```diff
@@ -1,38 +1,45 @@
-# How to contribute
-
-:fireworks: :balloon: **First off, thanks for taking the time to contribute!** :sparkler: :confetti_ball:
-
-We provide you a set of guidelines for contributing to Zowe&trade; documentation, which are hosted in the [Zowe / docs-site][b5d824b5] on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose content changes to this documentation.
-
-  [b5d824b5]: https://github.com/zowe/docs-site "https://github.com/zowe/docs-site"
-
-
-:arrow_right: [Before you get started](#start)
-
-:arrow_right: [Contributing to documentation](#contribute)
-
-:arrow_right: [Documentation style guide](#style)
-
-:arrow_right: [Word usage](#usage)
-
-
-## Before you get started<a name="start"></a>
-
-The Zowe documentation is written in Markdown markup language. Not familiar with Markdown? [**Learn the basic syntax**][84fff357].
-
-
-[84fff357]: https://www.markdownguide.org/basic-syntax "https://www.markdownguide.org/basic-syntax"
-
-## Contributing to documentation<a name="contribute"></a>
-
-You can use one of the following ways to contribute to documentation:
-
-- Send a GitHub pull request to provide a suggested edit for the content by clicking the **Edit this page** link at the top of each page.
-- Open an issue in GitHub by clicking the **Open doc issue** link at the top of each page to request documentation to be updated, improved, or clarified by providing a comment.
-
-### Sending a GitHub pull request
-
-You can provide suggested edit to any documentation page. After you make the changes, you submit updates in a pull request for the Zowe content team to review and merge.
+# Contributing to Zowe Documentation <!-- omit in toc -->
```

Comment:
@MichaelABroadcom can you please remove this comment `<!-- omit in toc -->`

Somehow Docusaurus ignores the comment in headings & renders it as a general text.
<img width="763" alt="Screenshot 2021-07-12 at 8 16 29 PM" src="https://user-images.githubusercontent.com/53327173/125308407-80a23380-e34e-11eb-9ebc-3bcb4a672ec9.png">

Reply:
Fixed